test/ZendViewRendererTest.php

```diff
@@ -607,6 +607,18 @@ public function testRenderChildWithDefaultParameter()
         static::assertEquals($content, $result);
     }
 
+    public function testCanRenderWithCustomSuffix()
```

Comment:
test name is a bit ambiguous as custom suffix could be specified as part of template name, please rename to
```suggestion
    public function testCanRenderWithCustomDefaultSuffix()
```